scss/_patterns_lists.scss

```diff
@@ -171,14 +172,14 @@ $list-status-icon-nudge: 0.3rem; // 0.3rem seems to be a magic number that posit
 
     .p-inline-list__item {
       @include vf-inline-list-item;
-
+      margin-right: $sph--small;
       position: relative;
 
       &::after {
         content: '\2022';
         line-height: map-get($line-heights, default-text);
-        position: absolute;
-        right: -$sp-unit;
+        position: relative;
+        right: -(math.div($sp-unit, 2));
```

Comment:
This solution works great, thanks!
Where does the `$sp-unit` / 2 value come from? Does it need to be exactly that, or just an arbitrary value to keep the dot centered? If it's not necesarily related to sp-unit maybe it should be hardcoded?

Reply:
it just centers the dot horizontally. It can be hardcoded. I did it like that because `$sp-unit` is used there.